tsl/test/expected/gapfill.out

```diff
@@ -1176,28 +1205,40 @@ SELECT
 FROM (VALUES (1),(2)) v(t)
 WHERE true AND true
 GROUP BY 1;
-ERROR:  invalid time_bucket_gapfill argument: finish cannot be NULL
+ERROR:  invalid time_bucket_gapfill argument: could not infer finish boundary from WHERE clause
 -- NULL finish and no usable time constraints
 SELECT
   time_bucket_gapfill(1,t,1,NULL)
 FROM (VALUES (1),(2)) v(t)
 WHERE true AND true
 GROUP BY 1;
-ERROR:  invalid time_bucket_gapfill argument: finish cannot be NULL
+ERROR:  invalid time_bucket_gapfill argument: could not infer finish boundary from WHERE clause
 -- expression with column reference on right side
 SELECT
   time_bucket_gapfill(1,t)
 FROM (VALUES (1),(2)) v(t)
 WHERE t > t AND t < 2
 GROUP BY 1;
-ERROR:  invalid time_bucket_gapfill argument: start cannot be NULL
+ERROR:  invalid time_bucket_gapfill argument: could not infer start boundary from WHERE clause
+-- expression with cast
+SELECT
+  time_bucket_gapfill(1,t1::int4)
```

Comment:
how does the expression with cast still work. I'm confused.

Reply:
fixed